frontend/app/templates/components/home-menu.hbs

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed"
+      data-toggle="collapse" data-target="#menu-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      {{ link-to 'Home' 'index' class="navbar-brand" }}
```

Comment:
@niranjan94 I might need some help with implementing the logout route. Or maybe it should be a globally available controller. 
All it needs to do is call '/logout' from the API, and transitionTo index upon success. Should I put the action in the ApplicationController , and use that??